sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -302,6 +303,71 @@ export interface ServiceBusConnectionStringProperties {
     sharedAccessSignature?: string;
 }
 
+// @public
+export class ServiceBusError extends MessagingError {
+    constructor(messagingError: MessagingError);
+    reason: ServiceBusErrorReason;
+}
+
+// @public
+export type ServiceBusErrorReason =
```

Comment:
I was thinking these codes would go for "code" ...and "reason" would have the elaborated list.

Reply:
This is how the .net error has it so if we want to differ we would need to coordinate with them.

Is there any reason that one vs the other is better? .code has always been totally free-form - in some cases it's even the direct code from the node.js error DNS error.